src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -1453,143 +1482,4 @@ int getIdealIndex()
-        implements PacketTransformer
-    {
-        /**
-         * {@inheritDoc}
-         */
-        @Override
-        public void close()
-        {
-            // no-op
-        }
-
-        /**
-         * {@inheritDoc}
-         */
-        @Override
-        public RawPacket[] reverseTransform(RawPacket[] pkts)
-        {
-            return pkts;
-        }
-
-        /**
-         * {@inheritDoc}
-         */
-        @Override
-        public RawPacket[] transform(RawPacket[] pkts)
-        {
-            if (ArrayUtils.isNullOrEmpty(pkts))
-            {
-                return pkts;
-            }
-
-            if (firstMediaMs == -1)
-            {
-                firstMediaMs = System.currentTimeMillis();
-            }
-
-            RawPacket[] extras = null;
-            for (int i = 0; i < pkts.length; i++)
-            {
-                if (!RTPPacketPredicate.INSTANCE.test(pkts[i]))
-                {
-                    continue;
-                }
-
-                long ssrc = pkts[i].getSSRCAsLong();
-
-                AdaptiveTrackProjection adaptiveTrackProjection
-                    = adaptiveTrackProjectionMap.get(ssrc);
-
-                if (adaptiveTrackProjection == null)
-                {
-                    pkts[i] = null;
-                    continue;
-                }
-
-                RawPacket[] ret;
-                try
-                {
-                    ret = adaptiveTrackProjection.rewriteRtp(pkts[i]);
-                }
-                catch (RewriteException ex)
-                {
-                    pkts[i] = null;
-                    continue;
-                }
-
-                if (ArrayUtils.isNullOrEmpty(ret))
-                {
-                    continue;
-                }
-
-                int extrasLen
-                    = ArrayUtils.isNullOrEmpty(extras) ? 0 : extras.length;
-                RawPacket[] newExtras = new RawPacket[extrasLen + ret.length];
-                System.arraycopy(ret, 0, newExtras, extrasLen, ret.length);
-
-                if (extrasLen > 0)
-                {
-                    System.arraycopy(extras, 0, newExtras, 0, extrasLen);
-                }
-
-                extras = newExtras;
-            }
-
-            return ArrayUtils.concat(pkts, extras);
-        }
-    }
-
-    /**
-     * The {@link PacketTransformer} that handles incoming/outgoing RTCP
-     * packets for this {@link BitrateController} instance. Internally,
-     * it delegates this responsibility to the appropriate media stream track
-     * bitrate controller ({@link AdaptiveTrackProjection}, etc).
-     */
-    private class RTCPTransformer
-        extends SinglePacketTransformerAdapter
-    {
-        /**
-         * Ctor.
-         */
-        RTCPTransformer()
-        {
-            super(RTCPPacketPredicate.INSTANCE);
-        }
-
-        /**
-         * {@inheritDoc}
-         */
-        @Override
-        public RawPacket transform(RawPacket pkt)
```

Comment:
Was this unused?

Reply:
I don't see it used anywhere in our branch, so I'll merge. We should see if it is actually needed, if it isn't there's a bunch of rtcp code in cc that we can remove